schemas/panels/markdown/markdown.cue

```diff
@@ -13,9 +13,10 @@
 
 package text
 
-#panel: {
+#panel: spec: plugin: {
```

Comment:
quite funny to see that that the cue tests are still working while the cue model and the json that is supposed to match the schema still pass.
I guess we can say the cue test doesn't work at all :(